src/options.rs

```diff
@@ -165,7 +179,7 @@ pub struct BuildOptions {
     /// Note, that in the second program, there are now 2 new coverage feedback points,
     /// and the fuzzer can store an input to the corpus at each condition that it passes;
     /// giving it a better chance of producing an input that reaches `res = 2;`.
-    pub disable_branch_folding: Option<bool>,
+    pub disable_branch_folding: bool,
```

Comment:
This was an Option<bool> because the default value was "true", now the default is "false". Add this to set the default to "true":

```rust
    #[arg(long, default_value_t = true)]
```

Reply:
done